common/create-playbook-service-catalog-item.adoc

```diff
@@ -19,6 +19,7 @@
 .. Assign the appropriate *Machine Credentials* from the drop-down list. 
 .. Add *Cloud* or *Network Credentials* from the drop-down lists. 
 .. Choose the *Host* against which to run the service item. 
+.. Set the *Max TTL* in minutes. The *Time To Live (TTL)* field allows you to set the maximum execution time for the playbook to run. 
```

Comment:
@cbudz Hey Chris, thanks for the addition. I think the brief description in the procedure reads well and provides enough info to the user.  By the way, I was able to see the **Max TTL (mins)** field, but not **Time To Live (TTL)**. If Time To Live (TTL) is not a field on the UI, I'd omit the mark up. Or, should that also read **Max TTL**?